### PR TITLE
About Page Design

### PR DESCRIPTION
**Pull request by mutahharaj**
Lists Websites about, mission, goal, and contributors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -97,6 +99,8 @@ const App = () => {
           {/* <Route path="/addbook" element={<AddBook/>}/> */}
           <Route path="/SearchResultsBook" element={<SearchResultsBook />} />
           <Route path="/ReaderQandA/:aid" element={<ReaderQandA />} />
+          
+          <Route path="/AboutPage" element={<AboutPage />} />
```
Comment (racecn) [inline src/App.js]: Can we make this route just /about or something similar?

File: src/components/AboutPage.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+export default function AboutPage () {
+    return (
+        <div className="flex flex-col h-screen bg-gray-100 ">
+            
+
+            <div className="flex-1 bg-maroon bg-opacity-80 flex flex-col justify-center items-center">
+                <h1 className='text-5xl text-pink-100 mt-16'>The Novelology Story</h1>
+                <h2 className='text-xl italic text-white mt-10 font-normal'>Turning pages, Uncovering Connections</h2>
+                <h2 className='text-xl italic text-white mt-10 font-normal'> </h2>
```
Comment (racecn) [inline src/components/AboutPage.js:18]: Are these headers for later use?

File: src/components/AboutPage.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+            <div className="flex-1 bg-gray-100 mt-10">
+                <div className="flex flex-row justify-center items-center w-full">
+                    
+                    <div className="flex flex-row justify-center items-center w-full">
+                        <div className="max-w-3xl">
+                            <div className="text-left mb-14">
+                                <h2 className="text-2xl font-normal text-maroon">About</h2>
+                                <p className="font-normal mt-2 text-xl text-justify">At Novelology, we're passionate about connecting avid readers through our social media book platform. Our platform offers numerous features designed to enhance your reading experience and foster lively discussions within the book community. Our inspiration stemmed from a desire to create a space where the book community could thrive.</p>
```
Comment (racecn) [inline src/components/AboutPage.js]: Can we link or talk more about the numerous features that we offer that another platform wouldn't?

**Review by racecn — CHANGES_REQUESTED** [review]

File: src/components/AboutPage.js (hunk 2 above)
Comment (mutahharaj) [reply src/components/AboutPage.js:18]: no, its just for this page

File: src/App.js (hunk 1 above)
Comment (mutahharaj) [reply src/App.js]: resolved this, i renamed it to /About

File: src/components/AboutPage.js (hunk 3 above)
Comment (mutahharaj) [reply src/components/AboutPage.js]: resolved it, i added a link to the home page per Rishitha

**Review by rishithab48 — APPROVED** [review]
Looks good!

File: src/components/AboutPage.js (hunk 2 above)
Comment (racecn) [reply src/components/AboutPage.js:18]: Ok, are they there for spacing? theres two empty header blocks

**Review by racecn — APPROVED** [review]

**Pull request merged**